includes/definitions/discovery/microsemipdsine.yaml

```diff
@@ -0,0 +1,25 @@
+mib: MICROSEMI-PDSINE-MIB
+modules:
+    sensors:
+        power:
+            data:
+                -
+                    oid: portTable
+                    value: portConsumptionPower
+                    num_oid: .1.3.6.1.4.1.7428.1.2.1.1.1.3.1.
+                    descr: portConsumptionPower
+                    index: 'portConsumptionPower.{{ $index }}'
+        voltage:
+            data:
+                -
+                    oid: mainPseTable
+                    value: mainVoltage
+                    num_oid: .1.3.6.1.4.1.7428.1.2.2.1.1.2.
+                    descr: System Voltage
```

Comment:
Can this have more than one entry as the descriptions would just be the same if so. If you can have more than one then do `descr: 'System Voltage {{ $index }}'`